assets/stylesheets/forms/_inputs.scss

```diff
@@ -0,0 +1,275 @@
+// Form Elements
+//
+// Weight: 6
+//
+// Style guide: FormElements
+
+// Inputs
+//
+// Markup:
+//  <div class="form-field">
+//   <label for="text">Test</label>
+//   <input type="text" placeholder="Placeholder" id="text" />
+//  </div>
+//  <div class="form-field">
+//   <label for="text" class="disabled">Test</label>
+//   <input type="text" placeholder=":disabled" disabled="disabled" id="text" />
+//  </div>
+//  <div class="form-field--has-errors">
+//   <label for="text">Test</label>
+//   <input aria-invalid="true" type="text" value="text" id="text" />
+//   <span>Something is wrong.</span>
+//  </div>
+//
+// Weight: 1
+//
+// Style guide: FormElements.Inputs
+
+$border-color: #29384A;
+$border-color-shadow: black;
+$placeholder-color: #888FA8;
+$focus-color: #2278F1;
+$focus-color-shadow: #CBE6FF;
+
+@mixin inputStyle{
```

Comment:
please use `dashes` for naming mixins and everyting CSS related. Javascript has `camelcase` as here.